Makefile

```diff
@@ -131,9 +89,17 @@ clean: ## Remove all generated files
 	rm -rf out/
 	rm -f cmd/clusterctl/examples/aws/provider-components-base-dev.yaml
 
-.PHONY: reset-bazel
-reset-bazel: ## Deep cleaning for bazel
-	bazel clean --expunge
+.PHONY: check-install
+check-install: ## Checks that you've installed this repository correctly
+	@./scripts/check-install.sh
+
+.PHONY: verify
+verify: ## Runs verification scripts to ensure correct execution
```

Comment:
Not sure what you mean by "ensure correct execution". Can you please clarify?

Reply:
It's just a generic statement :D which can be any execution, for example local development

Reply:
😃 ok!